server/src/index.ts

```diff
@@ -1,50 +1,50 @@
-import 'reflect-metadata'
-import { createConnection } from 'typeorm'
-import * as express from 'express'
-import * as bodyParser from 'body-parser'
-import * as cors from 'cors'
+import 'reflect-metadata';
+import { createConnection } from 'typeorm';
+import * as express from 'express';
+import * as bodyParser from 'body-parser';
+import * as cors from 'cors';
 
-import { Request, Response } from 'express'
-import { Routes } from './routes'
+import { Request, Response } from 'express';
+import { Routes } from './routes';
 
 createConnection()
     .then(async (connection) => {
-        const app = express()
-        app.use(bodyParser.json({ limit: '50mb' }))
+        const app = express();
+        app.use(bodyParser.json({ limit: '50mb' }));
         app.use(
             cors({
                 credentials: true,
                 origin: '*',
             })
-        )
+        );
 
-        app.use(express.urlencoded({ limit: '50mb', extended: true }))
+        app.use(express.urlencoded({ limit: '50mb', extended: true }));
 
         // register express routes from defined application routes
 
         Routes.forEach((route) => {
-            //эта функция автоматически создана при инициализации проекта typeORM
-            ;(app as any)[route.method](
+            //this function is automatically created when the typeORM project is initialized
+            (app as any)[route.method](
                 route.route,
                 (req: Request, res: Response, next: Function) => {
                     const result = new (route.controller as any)()[
                         route.action
-                    ](req, res, next)
+                    ](req, res, next);
                     if (result instanceof Promise) {
                         result.then((result) =>
                             result !== null && result !== undefined
                                 ? res.send(result)
                                 : undefined
-                        )
+                        );
                     } else if (result !== null && result !== undefined) {
```

Comment:
why don't you write like this `else if (result)`
what type can the variable result be?